__tests__/chart.test.js

```diff
@@ -0,0 +1,104 @@
+const PPTX = require('../index.js');
+const fs = require('fs');
+const tmpDir = `${__dirname}/tmp`;
+
+let { PptxUnitHelper } = require('../lib/helpers/unit-helper');
+
+describe('Presentation Module', () => {
```

Comment:
Better just to be chart module, since it just tests chart generation. Of course, the test hits a lot more places than the chart module itself (i.e. Presentation, factory classes, helpers, etc.).

Reply:
Done.